tensorcircuit/mpscircuit.py

```diff
@@ -781,18 +762,18 @@ def slice(self, begin: int, end: int) -> "MPSCircuit":
             nqubits,
             tensors=tensors,
             center_position=center_position,
-            **self.rules,
+            split=self.split.copy(),
         )
         return mps
 
     def expectation(
         self,
         *ops: Tuple[Gate, List[int]],
-        resue: bool = True,
+        reuse: bool = True,
         other: Optional["MPSCircuit"] = None,
         conj: bool = True,
         normalize: bool = False,
-        **rules: Any,
+        split: Dict[str, Any] = None,
```

Comment:
the type should be Optional[Dict] otherwise mypy complains type error?

Reply:
Just a intermediate push (I think I should not do this) so haven't checked the black, mypy, etc, will have more changes